### ISSUE TITLE
Theia debug session error notifications can provide an option to open 'launch.json' as it is done in VSCode

### ISSUE THREAD

**Issue by Bolodya1997** [eclipse-theia/theia#10912]
Theia debug session error notifications can provide an option to open 'launch.json' as it is done in VSCode.

How it is shown in Theia:
![image](https://user-images.githubusercontent.com/20160689/159685555-e77813d4-5781-40f3-8e59-2672f23f2c15.png)

How it is shown in VSCode:
![image](https://user-images.githubusercontent.com/20160689/159685651-68ce8c75-2afd-4ec8-a0fc-409f83d421ff.png)

### Feature Description:
Add <Open 'launch.json'> option for debug session launch error messages.

**Comment by vince-fugnitto** [comment]
@Bolodya1997 thank you for reporting the feature request, is it something you are interested in contributing?

**Comment by Bolodya1997** [comment]
@vince-fugnitto 
No, probably there is someone else who will find it useful. It seems to be a hard task for me :)
